assets/controllers/subject_controller.js

```diff
@@ -218,7 +218,7 @@ export default class extends Controller {
     async showModPanel(event) {
         event.preventDefault();
 
-        let container = this.element.querySelector('.moderate-inline')
+        let container = this.element.querySelector('.moderate-inline');
```

Comment:
sneaking in this ultra high priority change that I may or may not have forgotten about

Reply:
ULTRA HIGH PRIO! 🚨